src/value.rs

```diff
@@ -190,11 +216,12 @@ impl ValueCommitment {
         let hasher = pallas::Point::hash_to_curve("z.cash:Orchard-cv");
         let V = hasher(b"v");
         let R = hasher(b"r");
```

Comment:
#43 adds constants for these.

Reply:
I want to make sure we don't forget to change these to refer to the constants in #43. Please add a TODO or file an issue.